spec/lib/twingly/url/hasher_spec.rb

```diff
@@ -1,4 +1,4 @@
-require "spec_helper"
+SimpleCov.command_name(File.basename(__FILE__, ".rb"))
```

Comment:
> The command names can be seen in the footer in report:

Also in the spec run output: https://travis-ci.org/twingly/twingly-url/jobs/156254738